### PR TITLE
Singulo/Radiation Tweaks

### PR DESCRIPTION
**Pull request by Pangogie**
This is mostly intended as a temporary fix so the singularity will stop vaporizing people. Please adjust as necessary.

Added RadPerSecondBase so the singularity can change rads per level correctly based on the defined dps.
Renamed radiation suit/hood.
Radiation is now applied per 1s instead of 0.5s. (I think.)
Nerfed radiation storm damage to be less obnoxious. 8 --> 5

:cl: Pancake
- fix: The singularity will no longer vaporize engineers in 2 seconds.

### CODE REVIEW

**Review by ElectroJr — COMMENTED** [review]
The fills are gonna conflict with #6006

**Comment by Pangogie** [comment]
> The fills are gonna conflict with #6006

Fixed.

File: Content.Shared/Radiation/SharedRadiationStorm.cs [hunk 1]
```diff
@@ -10,6 +10,7 @@ public abstract class SharedRadiationPulseComponent : Component
     {
         public override string Name => "RadiationPulse";
 
+        public virtual float RadsPerSecondBase { get; set; }
```
Comment (ElectroJr) [inline Content.Shared/Radiation/SharedRadiationStorm.cs:13]: I don't think the client makes use of `RadsPerSecondBase`, so you can just leave that server-side?

File: Content.Shared/Radiation/SharedRadiationStorm.cs [hunk 2]
```diff
@@ -30,6 +31,7 @@ public abstract class SharedRadiationPulseComponent : Component
     [Serializable, NetSerializable]
     public class RadiationPulseState : ComponentState
     {
+        public readonly float RadsPerSecondBase;
```
Comment (ElectroJr) [inline Content.Shared/Radiation/SharedRadiationStorm.cs:34]:
```suggestion
```

File: Content.Server/Radiation/RadiationPulseComponent.cs [hunk 3]
```diff
@@ -49,6 +49,16 @@ public override bool Decay
...
         [DataField("dps")]
+        public override float RadsPerSecondBase
+        {
+            get => _radsPerSecond;
+            set
+            {
+                _radsPerSecond = value;
+                Dirty();
+            }
+        }
```
Comment (ElectroJr) [inline Content.Server/Radiation/RadiationPulseComponent.cs:60]: Both `RadsPerSecondBase` and `RadsPerSecond` are backed by the same value (`_radsPerSecond`). Is there supposed to be a `_radsPerSecondBase`?

File: Content.Shared/Radiation/SharedRadiationStorm.cs [hunk 4]
```diff
@@ -30,6 +31,7 @@ public abstract class SharedRadiationPulseComponent : Component
...
         public readonly float RadsPerSecond;
```
Comment (ElectroJr) [inline Content.Shared/Radiation/SharedRadiationStorm.cs:35]: Just adding this in here is not sufficient to make sure that this variable is networked.

You also need to add it to the constructor (see how `RadsPerSecond` is set a few lines below this), and you also need to then make use of it in `HandleComponentState()` in the client's `RadiationPulseComponent`.

Alternatively, if it just doesn't need to be networked (which seems to be the case, given that its only used to set `RadsPerSecond` which is networked), then you can just remove it from the state.

Comment (ElectroJr) [reply Content.Shared/Radiation/SharedRadiationStorm.cs:35]: And also remove the dirtying when setting.

I.e., the code from before:
```cs
public override float RadsPerSecondBase
{
    get => _radsPerSecond;
    set
    {
        _radsPerSecond = value;
        Dirty();
    }
}
```
could just be
```cs
public override float RadsPerSecondBase { get; set; }
```

**Review by ElectroJr — CHANGES_REQUESTED** [review]
Need to fix or remove the networking/component state stuff. and conflicts.

**Comment by Pangogie** [comment]
I just heckin' hardcoded in the radiation values for now. I'm not competent enough to do the proper C# necessary for the other methods. Again, this is just intended as a fix so the singulo stops doing 30+ dps and vaporizing engineers the second it starts.

**Review by ElectroJr — COMMENTED** [review]
Even if radiation needs to be re-done eventually, it should still be in yaml. Also instead of just deleting the changes, you deleted `engineer.yml` altogether.

If you just want to get singulo fixed quickly, mind if I add commits to this branch?

**Comment by Pangogie** [comment]
> Even if radiation needs to be re-done eventually, it should still be in yaml. Also instead of just deleting the changes, you deleted `engineer.yml` altogether.
> 
> If you just want to get singulo fixed quickly, mind if I add commits to this branch?

I did?! Gosh dang it. I thought I was just deleting the change when I did it through Github.

Ya go right for it, man. I'm not the best at C#.

**Review by ElectroJr — APPROVED** [review]
This isn't so much count as an "approve" as it is just a  "no longer request changes", given I made those changes, somone else should approve this.

**Pull request merged**